driver-core/src/main/com/mongodb/client/model/changestream/SplitEvent.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.changestream;
+
+import org.bson.codecs.pojo.annotations.BsonCreator;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+
+import java.util.Objects;
+
+/**
+ * The current fragment, out of the total number of fragments.
+ * When the change stream's backing aggregation pipeline contains the
+ * <code>$changeStreamSplitLargeEvent</code> stage, events larger than 16MB
+ * will be split into multiple events.
+ *
+ * @since 4.11
+ * @mongodb.server.release 7.0
```

Comment:
It was apparently backported to 6.0.9, but the fix version is 7 https://jira.mongodb.org/browse/SERVER-55062 . I will leave 7, since it was on the Jira for this ticket, but we can check with the team.